platforms/hyperledger-fabric/charts/invoke_chaincode/templates/invoke_chaincode.yaml

```diff
@@ -111,6 +115,34 @@ spec:
           echo "${CACERTS}" >> ${OUTPUT_PATH}/cacerts/ca.crt
           echo "${KEYSTORE}" >> ${OUTPUT_PATH}/keystore/server.key
           echo "${SIGNCERTS}" >> ${OUTPUT_PATH}/signcerts/server.crt
+          
+          ######################################################################
+
+          #Downloading Certificates for all admin users
```

Comment:
Downloading certificates is not required for fabric version 1.4.x, so this should be only execute for v2.x